package.json

```diff
@@ -34,41 +34,39 @@
     "ol": "^6.0.0"
   },
   "scripts": {
-    "build": "tsc -p tsconfig.json && npm run build:browser",
     "build:browser": "webpack --config browser-build.config.js",
-    "prebuild": "npm run test",
-    "pretest": "npm run lint",
+    "build": "tsc -p tsconfig.json && npm run build:browser",
+    "lint": "eslint -c .eslintrc.js --ext .ts && tsc --noEmit --project tsconfig.json",
     "prepublishOnly": "npm run build",
-    "test": "jest",
+    "release": "np --no-yarn && git push https://github.com/geostyler/geostyler-openlayers-parser.git master",
     "test:watch": "jest --watchAll",
-    "lint": "eslint -c .eslintrc.js --ext .ts && tsc --noEmit --project tsconfig.json",
-    "release": "np --no-yarn && git push https://github.com/geostyler/geostyler-openlayers-parser.git master"
+    "test": "jest --coverage"
```

Comment:
How about `npm run lint && jest --coverage`? This makes it easier to verify that your commit is valid locally. I would prefer this explicit syntax over using `pretest` because you might want to run the linting separately from test.

Also I think it is good practice to add a typecheck test. Could be as simple as `"typecheck": "tsc -p tsconfig.json --noEmit"`.